pkg/systemd/host.js

```diff
@@ -148,7 +148,8 @@ function ServerTime() {
         client.call(timedate.path,
                     "org.freedesktop.DBus.Properties", "Get", [ "org.freedesktop.timedate1", "NTPSynchronized" ]).
             fail(function (error) {
-                if (error.name != "org.freedesktop.DBus.Error.UnknownProperty")
+                if (error.name != "org.freedesktop.DBus.Error.UnknownProperty" &&
+                    error.message != "not-found")
```

Comment:
I think using ```error.problem``` would be better here. The message can change depending on who comes around and edits ```cockpit.message()```